project.py

```diff
@@ -147,12 +146,47 @@ def get_conture(self, image):
         hsv_image = cv.cvtColor(image, cv.COLOR_BGR2HSV)
         mask = self.get_mask(hsv_image)
         return self.find_board(mask)
-        
 
+    @timeit
+    def image_search(self, image):
+        """
+        Main image search function.
+        """
+        im = cv.imread(image, cv.CV_LOAD_IMAGE_COLOR)
+        self.create_trackbars()
+        while(True):
+            self.get_board(im)
+            if cv.waitKey(1) & 0xFF == 27:
+                break
+
+    @timeit
```

Comment:
@Foxiy this is nice and everything but the output is exteremely long once you use this for 3m long video.

I think we can safely remove it from here and only debug performance on images.